packages/salesforcedx-vscode-lwc/src/commands/commandUtils.ts

```diff
@@ -0,0 +1,25 @@
+import * as open from 'open';
```

Comment:
Instead of using the `open` module we should use vscode's `openExternal` api.

Reply:
nice